R/quosure.R

```diff
@@ -211,6 +217,13 @@ quo_label <- function(quo) {
 quo_text <- function(quo, width = 60L, nlines = Inf) {
   expr_text(quo_expr(quo), width = width, nlines = nlines)
 }
+#' @export
+quo_name <- function(quo) {
+  if (!is_symbol(quo)) {
+    abort("`quo` must quote a symbol")
```

Comment:
does this look right? https://github.com/hadley/rlang/pull/71/commits/2c10421c162ec509f63b5383f6a0892a13918797